packages/docusaurus-theme-search/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@markprompt/docusaurus-theme-search",
+  "version": "0.0.0",
+  "description": "A Markprompt search theme for Docusaurus",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/motifland/markprompt-js.git",
+    "directory": "packages/react"
+  },
+  "license": "MIT",
+  "sideEffects": false,
+  "type": "module",
+  "exports": "./dist/index.cjs",
+  "main": "dist/index.cjs",
+  "files": [
+    "dist",
+    "src/theme"
+  ],
+  "scripts": {
+    "prepack": "tsc --build && cpy 'src/**/*.css' dist"
+  },
+  "dependencies": {
+    "@docusaurus/types": "^2.0.0",
+    "@markprompt/core": "0.4.2",
```

Comment:
Do we need this explicitly, as `@markprompt/react` also has this as a dependency?

Reply:
Currently yes, because we re-export its `Options`.

I’ll switch to `RootProps`, which removes this need.